tests/queries/0_stateless/02421_json_decimals_as_strings.sql

```diff
@@ -0,0 +1,4 @@
+select toDecimal128(42.42, 5) as d format JSONEachRow settings output_format_json_quote_decimals=1;
```

Comment:
Only to Decimals, but it make sense to add new setting for `Float64` as it's done for 64-bit integers, because most libraries support only 32-bit numbers in json. I will add new setting